<ui version="4.0" >
 <class>ExportDialog</class>
 <widget class="QDialog" name="ExportDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>406</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <widget class="QTabWidget" name="tabWidget" >
     <property name="currentIndex" >
      <number>2</number>
     </property>
     <widget class="QWidget" name="saveTab_2" >
      <attribute name="title" >
       <string>Save</string>
      </attribute>
      <attribute name="toolTip" >
       <string>Save your work into an image</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4" >
       <item>
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>Your work will be saved to a picture. Please fill the following options:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3" >
         <property name="title" >
          <string>File properties</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" >
            <item>
             <widget class="QLabel" name="label" >
              <property name="text" >
               <string>Save file path:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="filePath" />
            </item>
            <item>
             <widget class="QPushButton" name="chooseFilePath" >
              <property name="text" >
               <string>Choose...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4" >
         <property name="title" >
          <string>Size and Orientation</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5" >
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" >
            <item>
             <widget class="QLabel" name="label_8" >
              <property name="text" >
               <string>Width</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="saveWidth" >
              <property name="maximum" >
               <number>999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" >
            <item>
             <widget class="QLabel" name="label_7" >
              <property name="text" >
               <string>Height</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="saveHeight" >
              <property name="maximum" >
               <number>999999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="saveLandscape" >
            <property name="text" >
             <string>Landscape</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="setBackground" >
            <property name="text" >
             <string>Set as system background</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveButton" >
         <property name="text" >
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="printTab_2" >
      <attribute name="title" >
       <string>Print</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6" >
       <item>
        <widget class="QGroupBox" name="groupBox" >
         <property name="title" >
          <string>Size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3" >
          <item>
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" >
             <widget class="QLabel" name="label_4" >
              <property name="text" >
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QDoubleSpinBox" name="printHeight" >
              <property name="maximum" >
               <double>10000.000000000000000</double>
              </property>
              <property name="value" >
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item rowspan="2" row="0" column="2" >
             <widget class="QComboBox" name="printUnity" >
              <item>
               <property name="text" >
                <string>pixels</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>cm</string>
               </property>
              </item>
              <item>
               <property name="text" >
                <string>inches</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QLabel" name="label_3" >
              <property name="text" >
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QDoubleSpinBox" name="printWidth" >
              <property name="maximum" >
               <double>10000.000000000000000</double>
              </property>
              <property name="value" >
               <double>0.000000000000000</double>
              </property>
             </widget>
            </item>
            <item row="2" column="1" >
             <widget class="QSpinBox" name="printDpi" >
              <property name="maximum" >
               <number>1000</number>
              </property>
              <property name="value" >
               <number>300</number>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="3" >
             <widget class="QCheckBox" name="printLandscape" >
              <property name="text" >
               <string>Print in landscape mode</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" >
             <widget class="QLabel" name="label_6" >
              <property name="toolTip" >
               <string>Dot Per Inch</string>
              </property>
              <property name="text" >
               <string>Dpi:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="printButton" >
         <property name="text" >
          <string>Print...</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="posterTab_2" >
      <attribute name="title" >
       <string>Print a poster</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7" >
       <item>
        <widget class="Wizard" native="1" name="m_posteRazorWizard" />
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="cancelButton" >
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
  <zorder>cancelButton</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Wizard</class>
   <extends>QWidget</extends>
   <header>3rdparty/posterazor/wizard.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
